go.mod

```diff
@@ -439,14 +439,9 @@ replace github.com/theupdateframework/go-tuf v0.3.1 => github.com/DataDog/go-tuf
 // excluded as DataDog/agent-payload/v5 v5.0.59 has been removed from source directory but is still present in proxy package
 exclude github.com/DataDog/agent-payload/v5 v5.0.59 // indirect
 
-// It appears that the v0.2.0 tag was modified.  Replacing with v0.2.1
-replace github.com/outcaste-io/ristretto v0.2.0 => github.com/outcaste-io/ristretto v0.2.1
-
 // openshift removed all tags from their repo, use the pseudoversion from the release-3.9 branch HEAD
 replace github.com/openshift/api v3.9.0+incompatible => github.com/openshift/api v0.0.0-20180801171038-322a19404e37
 
-replace github.com/cyphar/filepath-securejoin v0.2.3 => github.com/cyphar/filepath-securejoin v0.2.4
-
-replace github.com/sirupsen/logrus v1.9.0 => github.com/sirupsen/logrus v1.9.3
-
 replace github.com/opencontainers/runc v1.1.9 => github.com/opencontainers/runc v1.1.12
+
+replace github.com/docker/docker v24.0.7+incompatible => github.com/docker/docker v25.0.5+incompatible
```

Comment:
Upgrading major versions here may not be safe. Is the a matching major version you could use instead?